src/main/resources/config/application-artemis.yml

```diff
@@ -12,17 +12,17 @@ artemis:
         base: <base>
     jira:
         url: https://jirabruegge.in.tum.de
-        user: <username>    # e.g. ga12abc
-        password: <password>
+        user: ge56miw    # e.g. ga12abc
```

Comment:
I think you accidentally pushed the changes here. Can you revert them?

Reply:
True, silly mistake